SU2_CFD/include/solvers/CEulerSolver.hpp

```diff
@@ -291,18 +291,62 @@ class CEulerSolver : public CSolver {
              **ExtAverageKine,
              **ExtAverageOmega;
 
+  su2double  ***TurboVelocityIn,
+             ***TurboVelocityOut,
+             ***MachIn,
+             ***MachOut;
+             
   su2double  **DensityIn,
-             **PressureIn,
-             ***TurboVelocityIn,
+             **PressureIn,         
              **DensityOut,
              **PressureOut,
-             ***TurboVelocityOut,
              **KineIn,
              **OmegaIn,
              **NuIn,
              **KineOut,
              **OmegaOut,
-             **NuOut;
+             **NuOut,
+             **TotalStaticEfficiency,
+             **TotalTotalEfficiency,
+             **KineticEnergyLoss,
+             **TRadius,
+             **TotalPressureLoss,
+             **MassFlowIn,
+             **MassFlowOut,
+             **FlowAngleIn,
+             **FlowAngleIn_BC,
+             **FlowAngleOut,
+             **EulerianWork,
+             **TotalEnthalpyIn,
+             **TotalEnthalpyIn_BC,
+             **EntropyIn,
+             **EntropyOut,
+             **EntropyIn_BC,
+             **PressureRatio,
+             **TotalTemperatureIn,
+             **EnthalpyOut,
+             **VelocityOutIs,
+             **EnthalpyOutIs,
+             **EntropyGen,
+             **AbsFlowAngleIn,
+             **TotalEnthalpyOut,
+             **RothalpyIn,
+             **RothalpyOut,
+             **TotalEnthalpyOutIs,
+             **AbsFlowAngleOut,
+             **PressureOut_BC,
+             **TemperatureIn,
+             **TemperatureOut,
+             **TotalPressureIn,
+             **TotalPressureOut,
+             **TotalTemperatureOut,
+             **EnthalpyIn,
+             **TurbIntensityIn,
+             **Turb2LamViscRatioIn,
+             **TurbIntensityOut,
+             **Turb2LamViscRatioOut,
+             **NuFactorIn,
+             **NuFactorOut;
```

Comment:
Is there a more generic way to store these quantities? Maybe something like what I propose in #861 to cleanup the constructors / destructors of CEulerSolver and CNSSolver (the bit on the auxiliary type to store aerodynamic coefficients),

Reply:
Yeah I agree, I ll create a new Struct or a Class as a container for these quantities.
Perhaps we can come up also with a new file structure where we can put this classes. Do you have any suggestion?

Reply:
I think that @talbring wants to eventually move this post processing to the new output.
Until then maybe we could have a "tools" folder in "solvers", I think we have something similar in the output.

Reply:
@pcarruscag  I have created new set of classes under the folder structure "objectives/**". Now there is only a single object going around in the CEulerSolver, the CTurbomachineryPerformance.